pkg/statsd/parser.go

```diff
@@ -170,3 +190,22 @@ func (dp *DatagramParser) parseLine(line []byte) (*gostatsd.Metric, *gostatsd.Ev
 	}
 	return l.run(line, dp.namespace)
 }
+
+func (dp *DatagramParser) initDebugMode(ctx context.Context) {
+	if dp.debugMode {
+		dp.debugInitOnce.Do(func() {
+			dp.debugChan = make(chan []*gostatsd.Metric, debugChannelBufferSize)
+
+			go func() {
+				for {
+					select {
+					case <-ctx.Done():
+						return
+					case metrics := <-dp.debugChan:
+						fmt.Printf("[%s]: %s\n", time.Now().Format("2019-08-22 00:00:00"), metrics)
```

Comment:
The time format is wrong.

Reply:
... it doesn't actually matter once it's replaced with a proper logger, but worth investigating for the exposure to golangs time formatting.